templates/video/show.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello VideoController!
+{% endblock %}
+
+{% block body %}
+	<div class="container">
+		<div>
+			<div class="row">
+				<div class="col-lg-3">
+					<img class="image-show" src="{{ asset('build/images/' ~ video.posterUrl) }}" alt="{{ video.title }} poster">
+				</div>
+				<div class="col-lg-3">
+					<h1 class="mt-0">{{ video.title }}</h1>
+					<p>{{ video.description }}</p>
+				</div>
+				<div class="col-lg-6">
+					<h1>{{ video.title }}</h1>
+					<div class="video-container">
+						<video id="player" playsinline controls data-poster="{{ asset('build/images/' ~ video.posterUrl) }}">
+							<source src="{{ asset('build/videos/' ~ video.videoUrl) }}" type="video/mp4"/>
+						</video>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+	<script src="https://cdn.plyr.io/3.7.8/plyr.polyfilled.js"></script>
+	<script>
+		const player = new Plyr('#player');
+	</script>
```

Comment:
Why do you put your JS directly in your HTML ? You'd better use app.js for this or a whole new .js file